distributed/comm/core.py

```diff
@@ -238,7 +238,7 @@ def resolve_address(addr):
 
 
 @gen.coroutine
-def connect(addr, timeout=3, deserialize=True):
+def connect(addr, timeout=3, deserialize=True, ssl_options=None):
```

Comment:
I think it is more convenient for users (although having it behave like jupyter where both modes work would be even nicer).  

The main reason there is that you may not want it to be global for all dask runs, since there is presumably some overhead in the encryption.  Also not sure if there is a single config file or if that can be specified per job.

As a point of comparison, Apache Spark allows passing these as command line arguments to  `spark-submit`